DataStructuresMemories/ViewControllers/CustomTableViewCell.swift

```diff
@@ -13,7 +13,7 @@ class CustomTableViewCell: UITableViewCell {
 }
 
 extension CustomTableViewCell: ConfigurableCell {
-    func configure(_ model: DataStruct) {
+    func configure(_ model: DataStructProtocol) {
```

Comment:
If you configure cell with model, say `with` explicitly.
Otherwise seems like cell does configure model instead of being configured with model